TODO

```diff
@@ -0,0 +1,29 @@
+- Set permissions on the named pipe/Unix socket correctly. No need to let
+  random users spy on worktrees' activity to which they have no read access.
+
+- Skip reporting directories (meaning: make sure we know the type of a path)
+  - To do that, be smart and look whether a path refers to a file in a
+    subdirectory, then make sure that the subdirectory is marked as such
+  - Maybe over-report, and only mark known directories?
+  - Or not worry about this for now, it's okay to report directories, use
+    Trace2 to measure the impact of this
+
+- Exclude submodules somehow.
```

Comment:
For the moment, we do not need to worry about this. It only results in over-reporting, so it does not result in _incorrect_ results, but if anybody using submodules cares enough, that might be a valuable thing to do in the future.

Reply:
I'll add a caveat to the man page for the daemon.

Reply:
and an task to create submodule tests.